LICENSE-THIRD-PARTY

```diff
@@ -34,6 +34,38 @@ CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+---------------------------------------------------------------------------------------------------
+                                       BSD 2-Clause License
```

Comment:
BSD 3 which we use above is a superset of BSD 2; should this just be under that license? Is it worth the trouble figuring out if they can be merged?